fybrik/README.md

```diff
@@ -52,6 +52,13 @@ You will need a copy of the Fybrik repository (`git clone https://github.com/fyb
    ```bash
    kubectl get pods -n fybrik-blueprints
    ```
+---
+ > _NOTE:_ If you are using OpenShift cluster you will see that the deployment fails because OpenShift doesn't allow `privileged: true` value in `securityContext` field by default. Thus, you should add the service account of the module's deployment to the `privileged SCC` using the following command:
+```bash
+oc adm policy add-scc-to-user privileged system:serviceaccount:fybrik-blueprints:<SERVICE_ACCOUNT_NAME>
+```
+> Then, the doployment will be restarted and creating the pods in `fybrik-blueprints` namespace.
```

Comment:
doployment -> deployment 
How about: "Then, the deployment will be restarted and pods will be created in the `fybrik-blueprints` namespace."

Reply:
This looks like somebody else (user ?) will restart the deployment. Maybe "The deployment will restart the failed pods and the pods should start successfully."

Reply:
What about this: "Then, the deployment will be restarted automatically and will restart the failed pods in `fybrik-blueprints` namespace."?

Reply:
The Kubernenes deployment will not be restarted, when deployment cannot achieve the desire number of running pods, **it** periodically restarts the pods.

Reply:
Fixed.